python-backend/wolfram_client.py

```diff
@@ -61,11 +61,24 @@ def limit(expression: str) -> str:
         try:
             result = WolframClient.ask(query="limit of {}".format(expression), include_pod="Limit")
             # only want to return: queryresult -> pods[0] -> subpods
-
             return result["queryresult"]["pods"][0]["subpods"]
         except Exception as e:
             logger.error("Failed to obtain Wolfram API result", e)
 
+    @staticmethod
+    def raw(expression: str) -> str:
+        """
+        Query the Wolfram results API with a string expression
+        :param expression: Mathematical expression for which we would like to get more information
+        :return: Wolfram API response
+        """
+        try:
+            result = WolframClient.ask(query=f"{expression}")
+            # just pass the result through
+            return result["queryresult"]
+        except Exception as e:
+            logger.error("Failed to obtain Wolfram API result", e)
```

Comment:
Would it be valuable to log the expression or is that provided by the exception?

Reply:
Wolfram always returns the query in the response, so there is no need to explicitly include it here.